### ISSUE TITLE
Assigning switches bye moving them.

### ISSUE THREAD

**Issue by Poffertjesporem** [FrSky-OS/discussion#33]
For easy programing it would be nice to have the switch/slider/stick appear in the input line when operated, this to avoid scrolling trough all the options.

**Comment by janekx** [comment]
It work for me in case of flight mode selection, where it not work?

**Comment by Poffertjesporem** [comment]
Model (menu), output map.

**Comment by Poffertjesporem** [comment]
I have to check, just thought it be a nice feature. Could be in there al ready?

**Comment by Poffertjesporem** [comment]
Model (menu)/ Input map, does not support it.

**Comment by AloftHobbies** [comment]
I would consider this a must have option too. Needs to be universal on the radio.

**Comment by janekx** [comment]
Yes it should be same acros all radio.  About popup i am not sure,  i quite often use one switch to switch more function at one time.  Each model should be checked on ground and check the functionality.
